integration-tests/src/test/java/tachyon/worker/DataServerIntegrationTest.java

```diff
@@ -53,30 +53,45 @@
   private static final int WORKER_CAPACITY_BYTES = 1000;
   private static final int USER_QUOTA_UNIT_BYTES = 100;
 
-  private static final String UNUSED_TRANSFER_TYPE = "UNUSED";
+  // DataServer variations.
+  private static final String NETTY_DATA_SERVER = "tachyon.worker.netty.NettyDataServer";
+  private static final String NIO_DATA_SERVER = "tachyon.worker.nio.NIODataServer";
+
+  // Netty transfer variations.
+  private static final String MAPPED_TRANSFER = "MAPPED";
+  private static final String FILE_CHANNEL_TRANSFER = "TRANSFER";
+  private static final String UNUSED_TRANSFER = "UNUSED";
+
+  // Remote block reader variations.
+  private static final String TCP_BLOCK_READER = "tachyon.client.tcp.TCPRemoteBlockReader";
+  private static final String NETTY_BLOCK_READER = "tachyon.client.netty.NettyRemoteBlockReader";
```

Comment:
Could these constants be reused? (For example in `RemoteBlockInStreamIntegrationTests`)